src/secure_random.cr

```diff
@@ -1,19 +1,61 @@
 require "base64"
 require "openssl/lib_crypto"
 
+# The SecureRandom library is an interface for creating secure random values in different formats, 
+# including Base64 string, hexadecimal string, uuid and others.
+#
+# For example:
+# ```crystal
+# SecureRandom.base64 #=> LIa9s/zWzJx49m/9zDX+VQ==
+# SecureRandom.hex    #=> c8353864ff9764a39ef74983ec0d4a38
+# SecureRandom.uuid   #=> c7ee4add-207f-411a-97b7-0d22788566d6
+# ```
 module SecureRandom
+  # Generates a random base64 string.
+  #
+  # The argument `n` determines its size, which is around `n` * 4/3.
+  # A base64 string may contain A-Z, a-z, 0-9, +, /, and =.
+  # 
+  # ```crystal
+  # SecureRandom.base64(4) #=> fK1eYg==
+  # ```
   def self.base64(n = 16)
     Base64.strict_encode(random_bytes(n))
   end
 
+  # Same as `#base64` but is URL-safe.
+  #
+  # In addition, an extra argument `padding`(boolean) decides if padding `==` is generated.
```

Comment:
Yeah but I'm not sure if it should be explained here since it might be not quite related to the method itself. How about I explain it in this way:

```
# In addition, an extra argument `padding` decides if padding `=` is generated. 
# If the argument `n` is divided by 3 with no remainder, there will be no `=` and 
# the padding will make no effect. Otherwise, a number (equals to the remainder) of `=`
# will be attached to the end of the string.
# 
# ```crystal
# SecureRandom.urlsafe_base64          #=> MAD2bw8QaBdvITCveBNCrw
# SecureRandom.urlsafe_base64(8,true) #=> vvP1kcs841I=
# SecureRandom.urlsafe_base64(16,true) #=> og2aJrELDZWSdJfVGkxNKw==
# ```

```